packages/bot/src/components/run-panel.jsx

```diff
@@ -2,17 +2,20 @@ import { Button, Drawer, Tabs } from 'deriv-components';
 import classNames               from 'classnames';
 import React                    from 'react';
 import { IconInfoOutline }      from './Icons.jsx';
-import '../assets/sass/run-panel.scss';
+import Summary                  from './summary.jsx';
+import                               '../assets/sass/run-panel.scss';
 
 const onRunBotClick = () => {
-    console.log('run bot'); // eslint-disable-line no-console
+    Blockly.BLOCKLY_CLASS_OLD.run();
```

Comment:
In `/sractch/index.js` under run function, please change `Blockly.mainWorkspace` to `Blockly.derivWorkspace`.